src/types/requests/HttpChargingStationRequest.ts

```diff
@@ -101,17 +102,20 @@ export interface HttpChargingStationOcppParametersRequest {
   forceUpdateOCPPParamsFromTemplate: boolean;
 }
 
-export interface HttpChargingStationSetMaxIntensitySocketRequest extends HttpChargingStationCommandRequest {
```

Comment:
I think we can keep HttpChargingStationCommandRequest with the chargingStationID right?
And all Action HTTP Request classes should implement it.

Reply:
all extends now :)